app/controllers/api/v1/tokens_controller.rb

```diff
@@ -127,6 +137,11 @@ def regenerate
         resource: token,
       )
 
+      # expire session if we're regenerating the current token
+      if token == current_token
+        cookies.delete(:session_id, domain: Keygen::DOMAIN, same_site: :none)
```

Comment:
Do we _want_ to expire the session when a token is regenerated?

Reply:
Or should we revoke all sessions and generate a new session entirely? I lean towards that.

Reply:
`regenerate!` can return the new session.